packages/skill-lesson/video/video-transcript.tsx

```diff
@@ -8,7 +8,8 @@ import {getTranscriptComponents} from '../markdown/transcript-components'
 
 export const VideoTranscript: React.FC<{
   transcript: string | any[]
-}> = ({transcript}) => {
+  noTitle?: boolean
```

Comment:
add this condition so that we are able to get Transcript title customized (with icon for example).